src/transaction/log_manager.c

```diff
@@ -14137,47 +14137,59 @@ cdc_get_start_point_from_file (THREAD_ENTRY * thread_p, int arv_num, LOG_LSA * r
     }
   else
     {
-      LOG_ARCHIVE_CS_ENTER (thread_p);
-      aligned_hdr_pgbuf = PTR_ALIGN (hdr_pgbuf, MAX_ALIGNMENT);
+      if (log_Gl.archive.vdes != NULL_VOLDES && log_Gl.archive.hdr.arv_num == arv_num)
+	{
+	  /* if target archive log volume is currenty mounted, then use that */
+	  process_lsa.pageid = log_Gl.archive.hdr.fpageid;
+	  process_lsa.offset = 0;
+	}
```

Comment:
1. I don't know well, but please check whether it needs `LOG_ARCHIVE_CS_ENTER()` because it seems that `vdes` and `hdr` of `log_Gl.archive` are changed not atomically in other threads.
2. What happens if `log_Gl.archive.vdes` is dismounted after this in another thread? Is it OK?

Reply:
Oh.. yes, I will check it and move the LOG_ARCHIVE_CS_ENTER() and EXIT(). thank you.

Reply:
1. information in log_Gl.archive is protected by LOG_ARCHIVE_CS_ENTER()/EXIT() 
2. After getting fpageid of archive log volume, it doesn't matter if it is dismounted. 
    This is because it is mounted again with the corresponding fpageid in the logpb_fetch_page() below.